gdp/column/gdp_column.py

```diff
@@ -554,64 +549,71 @@ def feed_vap_enthalpy_expr(_, c):
     def feed_vap_enthalpy_calc(_, c):
-        return m.H_V_spec_feed[c] == m.feed_vap_enthalpy_expr[c]
+        return m.H_V_spec_feed[c] == m.feed_vap_enthalpy_expr[c]  # Constraint for feed vapor enthalpy
+
 
 
 def _build_condenser_energy_balance(m):
     t = m.condens_tray
 
+    # Energy balance for partial condenser
     @m.partial_cond.Constraint()
     def partial_condenser_energy_balance(_):
         return -m.Qc + sum(
             - m.D[c] * m.H_L[c, t]  # heat of liquid distillate
             - m.L[c, t] * m.H_L[c, t]  # heat of liquid to tray below
             + m.V[c, t - 1] * m.H_V[c, t - 1]  # heat of vapor from tray below
             - m.V[c, t] * m.H_V[c, t]  # heat of vapor from partial condenser
-            for c in m.comps) * 1E-3 == 0
+            for c in m.comps) * 1E-3 == 0  # Ensuring net heat in partial condenser is zero (equilibrium)
 
+    # Energy balance for total condenser
     @m.total_cond.Constraint()
     def total_condenser_energy_balance(_):
         return -m.Qc + sum(
             - m.D[c] * m.H_L[c, t]  # heat of liquid distillate
             - m.L[c, t] * m.H_L[c, t]  # heat of liquid to tray below
             + m.V[c, t - 1] * m.H_V[c, t - 1]  # heat of vapor from tray below
-            for c in m.comps) * 1E-3 == 0
+            for c in m.comps) * 1E-3 == 0  # Ensuring net heat in total condenser is zero (equilibrium)
 
+    # Constraints to calculate enthalpy for liquid and vapor based on temperature for condenser
     @m.Constraint(m.comps)
     def condenser_liq_enthalpy_calc(_, c):
-        return m.H_L[c, t] == m.liq_enthalpy_expr[t, c]
+        return m.H_L[c, t] == m.liq_enthalpy_expr[t, c]  # Liquid enthalpy = f(T)
 
     @m.partial_cond.Constraint(m.comps)
     def vap_enthalpy_calc(_, c):
-        return m.H_V[c, t] == m.vap_enthalpy_expr[t, c]
+        return m.H_V[c, t] == m.vap_enthalpy_expr[t, c]  # Vapor enthalpy = f(T)
 
 
 def _build_reboiler_energy_balance(m):
     t = m.reboil_tray
 
+    # Energy balance for reboiler
     @m.Constraint()
     def reboiler_energy_balance(_):
         return m.Qb + sum(
             m.L[c, t + 1] * m.H_L[c, t + 1]  # Heat of liquid from tray above
             - m.B[c] * m.H_L[c, t]  # heat of liquid bottoms if reboiler
             - m.V[c, t] * m.H_V[c, t]  # heat of vapor to tray above
-            for c in m.comps) * 1E-3 == 0
+            for c in m.comps) * 1E-3 == 0  # Ensuring net heat in reboiler is zero (equilibrium)
 
+    # Constraints to calculate enthalpy for liquid and vapor based on temperature for reboiler
     @m.Constraint(m.comps)
     def reboiler_liq_enthalpy_calc(_, c):
-        return m.H_L[c, t] == m.liq_enthalpy_expr[t, c]
+        return m.H_L[c, t] == m.liq_enthalpy_expr[t, c]  # Liquid enthalpy = f(T)
 
     @m.Constraint(m.comps)
     def reboiler_vap_enthalpy_calc(_, c):
-        return m.H_V[c, t] == m.vap_enthalpy_expr[t, c]
+        return m.H_V[c, t] == m.vap_enthalpy_expr[t, c]  # Vapor enthalpy = f(T)
```

Comment:
What do you mean with f(T)?

Reply:
I might wrote incorrectly considering solely as the only function of Temperature.

Reply:
Please do not use the notation of f(T) as we have many functions in the code. Use rather words to explain as in the last comments.